packages/attention/stories/Attention.stories.tsx

```diff
@@ -1,48 +1,62 @@
-import React from "react";
-import { Attention } from "../src";
-import { Box } from "../../box/src";
-import { Button } from "../../button/src";
+import React from 'react'
+import { Attention } from '../src'
+import { Box } from '../../box/src'
+import { Button } from '../../button/src'
 
-const metadata = { title: "Overlays/Attention" };
-export default metadata;
+const metadata = { title: 'Overlays/Attention' }
+export default metadata
 
 export function Callout() {
   return (
-    <div className="flex items-center">
+    <div className='flex items-center'>
       <Box info>
-        <h1>I am a box full of info</h1>
+        <h1 aria-details="callout-bubbletext">I am a box full of info</h1>
+      </Box>
+      <Attention callout placement='right' isShowing={true}>
+        <p id="callout-bubbletext" style={{ width: 200 }}>
+          I'm a callout because that box over there is new or something
+        </p>
+      </Attention>
+    </div>
+  )
+}
+export function CalloutResettingRoleAndAriaLabel() {
+  return (
+    <div className='flex items-center'>
+      <Box info>
+        <h1 aria-details="callout-reset-bubbletext">I am a box full of info</h1>
       </Box>
       <Attention
         callout
-        placement="right"
-        ariaLabel="Green speech bubble pointing left"
-        ariaRole="img"
+        role=''
+        aria-label=''
+        placement='right'
         isShowing={true}
       >
-        <p style={{ width: 200 }}>
-          I'm a callout because that box over there is new or something
+        <p id="callout-reset-bubbletext" style={{ width: 200 }}>
```

Comment:
I _think_ that in order for this to work, you'd need to set `role='tooltip'` or `role='img'` on this element. Otherwise the screen reader won't know that it should read this text. I'd also add `pointing left` at the end of the example in line 37, just to display that we should not forget to explain what the component looks like.

Reply:
Ah you're probably right! I guess I should add `role='tooltip`' or `role='img'` for the `p`-element in the reset-example in the vue-PR as well then? :)